test/extensions/geoip_providers/maxmind/geoip_provider_test.cc

```diff
@@ -177,7 +192,7 @@ TEST_F(GeoipProviderTest, ValidConfigCityAndIspDbsSuccessfulLookup) {
         city: "x-geo-city"
         asn: "x-geo-asn"
     city_db_path: "{{ test_rundir }}/test/extensions/geoip_providers/maxmind/test_data/GeoLite2-City-Test.mmdb"
-    isp_db_path: "{{ test_rundir }}/test/extensions/geoip_providers/maxmind/test_data/GeoLite2-ASN-Test.mmdb"
```

Comment:
If `isp_db_path` still exists, shouldn't this be additional tests rather than replacing `isp_db_path` with `asn_db_path`? (i.e. it looks like this change has made it so `isp_db_path` has less test coverage than it did before.)

Reply:
Ah, never mind, I see the important part does have a new test.